ingesters/http.md

```diff
@@ -503,3 +503,9 @@ Log-File="/opt/gravwell/log/http_ingester.log"
 Health-Check-URL="/logbot/are/you/alive"
 
 ```
+
+## Timestamp Formats
+
+The HEC data format expects the timestamps to be Unix timestamps in seconds formatted as an integer or optionally a floating point number; the limited documentation also states that the timestamp should be encoded in the JSON object as a number.  The Gravwell HEC compatible ingester will make a best effort to extract and process timestamps generated by data providers that communicate with HEC endpoints, however if a timestamp cannot be interpreted at all the endpoint will return a 400 response rather than attempt to process a potentially invalid timestamp.
```

Comment:
```suggestion
The HEC data format expects the timestamps to be Unix timestamps in seconds formatted as an integer or optionally a floating point number; the limited documentation also states that the timestamp should be encoded in the JSON object as a number.  The Gravwell HEC compatible ingester will make a best effort to extract and process timestamps generated by data providers that communicate with HEC endpoints However, if a timestamp cannot be interpreted at all, the endpoint will return a 400 response rather than attempt to process a potentially invalid timestamp.
```